R/sequential.R

```diff
@@ -0,0 +1,94 @@
+#' Perform multiple requests in sequence
+#'
+#' Given a list of requests, this function performs each in turn, returning
+#' a list of responses. It's slower than [req_perform_parallel()] but
+#' has fewer limitations.
+#'
+#' @inheritParams req_perform_parallel
+#' @inheritParams req_perform_iteratively
+#' @param on_error What should happen if one of the requests throws an
```

Comment:
@mgirlich let me know what you think of this error handling, and if it looks good, I'll port to `req_perform_parallel()`.

Reply:
And `req_perform_iterative()`, although that won't have the `"continue"` option.